addons/l10n_ar_withholding/wizards/account_payment_register.py

```diff
@@ -82,77 +85,28 @@ def _compute_withholdings(self):
             rec.withholding_ids =[Command.clear()] + [Command.create({'tax_id': x.id}) for x in taxes]
         (self - supplier_recs).withholding_ids = False
 
-    @api.depends('withholding_ids.amount', 'amount')
+    @api.onchange('l10n_latam_check_id')
+    @api.depends('withholding_ids.amount', 'amount', 'l10n_latam_check_id')
     def _compute_net_amount(self):
         for rec in self:
             if rec.l10n_latam_check_id:
                 rec.net_amount = rec.l10n_latam_check_id.amount
+                base_lines = rec.line_ids.move_id.invoice_line_ids
+                conversion_rate = rec._get_conversion_rate()
```

Comment:
conversion_rate is normally not needed (but why might have a check to respect the right currency)

Reply:
this was refactored